polaris-tokens/src/themes/base/color.ts

```diff
@@ -1010,6 +1016,26 @@ export const color: {
     description:
       'The active state (on press) color for borders on an inverse background.',
   },
+  'color-tooltip-border-tail-down-experimental': {
+    value: colors.gray[9],
+    description: 'The border color for tool tip tails pointing down.',
+  },
+  'color-tooltip-border-tail-up-experimental': {
+    value: colors.gray[8],
+    description: 'The border color for tool tip tails pointing up.',
+  },
+  'color-border-gradient-experimental': {
+    value: `linear-gradient(to bottom, ${colors.gray[7]}, ${colors.gray[10]} 78%, ${colors.gray[11]})`,
+  },
```

Comment:
all the values below are the same since we don't have designs for the light border gradients yet. the dark experimental theme overrides these with values that will be used for the top bar

Reply:
Makes sense to me 👍 Every token must be referenced in our base theme to derive the types and validate overrides